sig-scalability/README.md

```diff
@@ -35,18 +35,28 @@ The Chairs of the SIG run operations and processes governing the SIG.
 
 The following subprojects are owned by sig-scalability:
 - **kubernetes-scalability-definition**
+  - Description: [Described below](#kubernetes-scalability-definition)
```

Comment:
Under this can you also mention the `configs-and-limits` directory? I think it's relevant here.

Reply:
It is (and was already described in readme).
But there wasn't owners file for that subdirectory.

I created https://github.com/kubernetes/community/pull/3054 to address that.